scripts/download_shellcheck.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+# Update shellcheck to the latest by downloading the binary (0.7.0 instead of default 0.4)
```

Comment:
the latest version at the moment is 0.7.0, but it might change soon. The same with the default version on Github Actions. So lets take 
out this part of he comment: (0.7.0 instead of default 0.4)

Reply:
Removed the last bit of the comment